tasks/pretest.js

```diff
@@ -52,6 +53,16 @@ function makeTestImageFolders() {
     makeOne(constants.pathToTestImagesDiff, 'test image diff folder');
 }
 
+// Make script file that define plotly in a RequireJS context
+function makeRequireJSFixture() {
+    var bundle = fs.readFileSync(constants.pathToPlotlyDistMin, 'utf-8'),
```

Comment:
**Important** I chose to test Require.js import with the `dist/plotly.min.js` bundle only.

That means that Require.js will only be tested on `npm version` commits (when the `dist/` is updated). 

Main reason for this shotcut: minifying the full plotly.js bundle take about 1 minute on CI.